apps/backoffice-v2/src/common/hooks/useHomeLogic/useHomeLogic.ts

```
@@ -1,39 +1,174 @@
+          color: getRiskColor(curr.riskLevel),
+        },
+      }),
+      {},
+    );
+  }, [currentStats]);
+
+  const approvedRiskConfig = useMemo(() => {
+    if (!currentStats) return {};
+    return currentStats.approvedCasesByRisk.reduce(
+      (acc, curr) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)